README.md

```diff
@@ -1,2 +1,138 @@
 # mitxpro-theme
-an open edX theme for MIT xPRO 
+An open edX theme for MIT xPRO 
+
+## Table of contents:
+1. Configure Open edX
+2. Configuring MIT xPRO theme
+
+
+### 1. Configure Open edX
+
+Following steps are inspired by [edx-devstack](https://github.com/edx/devstack).
+
+#### Clone edx/devstack
+
+```
+$ git clone https://github.com/edx/devstack
+$ cd devstack
```

Comment:
any branch specific to `ironwood.master` for devstack ?

Reply:
So after struggling alot with devstack. I missed devstack to be `ironwood.master`.  That was the key :)

Reply:
I think, that would be the change that need to be update.